SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLoadBalancingHandler.cpp

```diff
@@ -292,3 +293,46 @@ VirtualWorkerId FSpatialLoadBalancingHandler::GetWorkerId(const AActor* NetOwner
 	}
 	return NewAuthVirtualWorkerId;
 }
+
+void FSpatialLoadBalancingHandler::UpdateActorsHandedOver(USpatialNetDriver& InNetDriver,
+														  const TSet<Worker_EntityId_Key>& EntitiesHandedOver,
+														  const TSet<AActor*>& ActorsHandedOver)
+{
+	const bool bStrategyWorkerEnabled = USpatialStatics::IsStrategyWorkerEnabled();
```

Comment:
minor: could be static maybe?

Reply:
can break things when running in editor - so will leave as is